modules.d.ts

```diff
@@ -130,6 +131,25 @@ export declare const MsgPack: unknown;
  */
 export declare const RealtimePresence: unknown;
 
+/**
+ * The module is experimental and the API is subject to change.
+ *
+ * Provides a {@link BaseRealtime} instance with the ability to interact with the experiemental channel groups feature.
+ *
+ * To create a client that includes this module, include it in the `ModulesMap` that you pass to the {@link BaseRealtime.constructor}:
+ *
+ * ```javascript
+ * import { BaseRealtime, WebSocketTransport, FetchRequest, RealtimePresence, ChannelGroups } from 'ably/modules';
+ * const realtime = new BaseRealtime(options, { WebSocketTransport, FetchRequest, RealtimePresence, ChannelGroups });
```

Comment:
I thought in this release we'll pack `ChannelGroups` in default web client, so SDK user won't have to declare WebSocketTransport, FetchRequest, RealtimePresence

Reply:
it is part of DefaultRealtime (import from `ably`) - part of default web client bundle
https://github.com/ably/ably-js/pull/1661/files#diff-5dd5bf8949d722411724b640db79f4d05ef91185d5b5c6e26af4b642520d2839R41

this are just typings for the modular variant